tracer/src/Datadog.Trace/AppSec/AttackerFingerprint/AttackerFingerprintHelper.cs

```diff
@@ -24,10 +24,16 @@ public static void AddSpanTags(Span span, IResult result)
             return;
         }
 
-        var securityCoordinator = new SecurityCoordinator(Security.Instance, span);
+        var securityCoordinator = SecurityCoordinator.TryGet(Security.Instance, span);
+
+        if (securityCoordinator is null)
+        {
+            Log.Warning("Security coordinator could not be instantiated, probably because of httpcontext null. AttackerFingerprintHelper.AddSpanTags won't be run");
```

Comment:
Note that you're already logging a warning in this scenario when you call `TryGet` - it's probably unnecessary to do both?

Reply:
fixed in [2acfc80](https://github.com/DataDog/dd-trace-dotnet/pull/6143/commits/2acfc808df89449a85d5b892ebd663217f86a139) thank you